pyproject.toml

```diff
@@ -55,97 +55,99 @@ dev = ["mne[doc,test]", "rcssmin"]
 # Dependencies for building the documentation
 doc = [
   "graphviz",
-  "intersphinx_registry>=0.2405.27",
-  "ipython!=8.7.0",
+  "intersphinx_registry >= 0.2405.27",
+  "ipython != 8.7.0",  # is it actually needed for docs?  also in "full-no-qt" and "test"
```

Comment:
Yes for syntax highlighting to work ipython needs to be installed IIRC. Incidentally I think this is where the `!=` pin comes from as well!